handlers/issue.go

```diff
@@ -28,44 +27,42 @@ func (s *Server) handleIssueCommentEvent(body []byte) {
 		glog.Errorf("fail to unmarshal: %v", err)
 	}
 	glog.Infof("prc: %v", prc)
-	comment := *prc.Comment.Body
 
-	// https://github.com/islinwb/test/pull/1
-	prID := strings.SplitAfter(prc.Issue.PullRequestLinks.GetHTMLURL(), "github.com/")[1]
-	// https://github.com/islinwb/test/pull/1.patch
-	// From <commit ID> MON ...
-	patchURL := prc.Issue.PullRequestLinks.GetPatchURL()
-	resp, err := http.Get(patchURL)
+	owner := *prc.Repo.Owner.Login
+
+	ctx := context.Background()
+	user := prc.Sender.Login
+	list, _, err := client.Repositories.ListCollaborators(ctx, owner, *prc.Repo.Name, nil)
+	fmt.Println("list", list)
 	if err != nil {
-		fmt.Println(err)
+		glog.Fatal("Cannot List the Collaborators", err)
 	}
 
-	resp1, err := ioutil.ReadAll(resp.Body)
+	comment := *prc.Comment.Body
+
+	assignees := strings.TrimPrefix(comment, "/assign @")
+	assign, _, err := client.Repositories.IsCollaborator(ctx, owner, *prc.Repo.Name, *user)
 	if err != nil {
-		fmt.Println(err)
-	}
+		glog.Fatal("Not the collaborator", err)
 
-	patchDetail := string(resp1)
-	reg := regexp.MustCompile(`From [A-Za-z0-9]{40}`)
-	commitIDstr := reg.FindString(patchDetail)
-	commitID := strings.TrimPrefix(commitIDstr, "From ")
+	}
+	reg := regexp.MustCompile("(?mi)^/(un)?assign(( @?[-\\w]+?)*)\\s*$")
```

Comment:
Just 1 suggestion, this assign logic will be common across issueComment and PR, lets try to find a way to keep such logic common. Something like client.X.AddAssignees... get X maybe through parameter.
This way we will reduce duplicate code